brokers/unified/broker.go

```diff
@@ -253,7 +260,7 @@ func sortMetas(metas []model.PluginMeta) (che []model.PluginMeta, vscode []model
 func getRegistryURL(plugin model.PluginFQN, defaultRegistry string) (string, error) {
 	var registry string
 	if plugin.Registry != "" {
-		registry = strings.TrimSuffix(plugin.Registry, "/")
+		registry = strings.TrimSuffix(plugin.Registry, "/") + "/plugins"
```

Comment:
Adding `/plugins` should be unnecessary, no? I'd vote against modifying user input.

Reply:
Agree, that can be set explicitly if needed.

Reply:
I think it's more question to Che Plugin Registry, 
@benoitf suggested to make root return README.md, and `/plugins/`->  `index.json`.
Adding `plugins` is a kind of Plugin Registry API. If we stop using `plugins`, then I think it's needed to rework Plugin Registry and make root request return `index.json`. It's my personal feeling and it may be wrong.

About settings `plugins` explicitly by user:
it may be OK but ATM it's not clear enough like a user deploys his custom plugin registry based on our, retrieves Plugin Registry Route URL from OpenShift console and it's logical to use it as `registryUrl` without `/plugins` segment.

If we change this behavior, then as Alex commented https://github.com/eclipse/che/issues/13104#issuecomment-483245433, we should consider making the corresponding changes for `UD`, `WsMaster`  and `Theia`.